changelog/1.7.0.md

```diff
@@ -1,21 +1,66 @@
 # Scala.meta 1.7.0
 
+## Cross-compilation against Scala.js
+
+It is our great pleasure to announce that the overwhelming majority of scala.meta modules are now supported by Scala.js!
+In 1.7.0, it is possible to do advanced tasks that include parsing, quasiquoting and prettyprinting in a browser.
+The only unsupported modules are `scalahost` and `testkit`.
+
+## Parser for Scaladoc comments
+
+## Semantic API
+
+- `scalahost` has been split into `scalahost`, a library that provides a scalac-based implementation of `Mirror`,
```

Comment:
Should 'Mirror' be described/explained a little? Perhaps I'm not a good sample of the target audience, but fwiw I have no understanding of what 'Mirror' means. Maybe I've just been out of the loop.